core-bundle/src/Security/Voter/DataContainer/ContentCompositionVoter.php

```diff
@@ -55,6 +55,12 @@ public function vote(TokenInterface $token, $subject, array $attributes): int
             return self::ACCESS_ABSTAIN;
         }
 
+        // If the record has an ID, it is most likely an undo operation, which should
+        // allowed if there were articles on a page without content composition.
+        if ($subject instanceof CreateAction && $subject->getNewId()) {
```

Comment:
As discussed in the Contao call, the restore operation should be an update action instead of a create action.